src/components/utils/inputs/input-hooks.js

```diff
@@ -490,6 +490,10 @@ export const useValidNodeName = ({ studyUuid, defaultValue, triggerReset }) => {
         },
         [studyUuid, intl, defaultValue, snackError]
     );
+    const debouncedValidName = useMemo(
```

Comment:
useCallback

Reply:
You must take the name parameter here as well

Reply:
Can you clear this point ? `validName` is not enough ?

Reply:
my bad and useMemo is better you are right